app/src/main/java/me/guillaumin/android/osmtracker/activity/TrackDetail.java

```diff
@@ -144,6 +148,21 @@ protected void onResume() {
 		map.put(ITEM_VALUE, Integer.toString(t.getTpCount()));
 		data.add(map);
 
+		// Distance
+		map = new HashMap<String, String>();
+		map.put(ITEM_KEY, getResources().getString(R.string.trackmgr_distance));
+		map.put(ITEM_VALUE, TracklistAdapter.distanceToString(stat.totalLength(), getResources()));
+		data.add(map);
+
+		// Speed
+		map = new HashMap<String, String>();
+		map.put(ITEM_KEY, getResources().getString(R.string.trackdetail_speed));
+		map.put(ITEM_VALUE, TracklistAdapter.speedToString(stat.averageSpeed(), getResources()) + " " +
```

Comment:
For those it would be preferable to use a placeholder as we do [here](https://github.com/nguillaumin/osmtracker-android/blob/master/app/src/main/java/me/guillaumin/android/osmtracker/view/VoiceRecDialog.java#L130) because in some languages that may not be expressed as "12 km" but perhaps "km 12"...